evm_loader/program/src/executor_state.rs

```diff
@@ -276,11 +276,11 @@ impl ExecutorSubstate {
             return Some(*value);
         }
 
-        if let Some(account) = self.accounts.get(&address) {
-            if account.reset {
-                return Some(H256::default());
-            }
-        }
```

Comment:
Imagine you have something stored in the parent state storage. 
In this case it is still be accessible after account reset.

Reply:
logic changed